src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
```

Comment:
Linking a SCSS file directly in HTML is not correct, as browsers do not understand SCSS syntax. You should compile SCSS to CSS and link the compiled CSS file instead.